src/lib/authorization.middleware.ts

```diff
@@ -0,0 +1,41 @@
+import { NextFunction, Request, Response } from "express";
+import { errorResponse } from "./responses/error.response";
+import { developmentAuthorize } from "./development.authorize";
+import { oktaAuthorization } from "./okta.authorize";
+
+/**
+ * @description - Authorization middleware to filter invalid requests.
+ *
+ * @param req
+ * @param res
+ * @param next
+ */
+function authorizationMiddleware(req: Request, res: Response, next: NextFunction): void {
+    const { headers } = req;
+    const { authorization } = headers;
+    const argv = process.argv?.slice(2);
```

Comment:
this is very brittle, if we added more args and put them in the wrong order we could deploy Airlock in the cloud and have it bypass OKTA.
instead of using `--prod` for switching between dev and prod auth, we should always use the same path (always call oktaAuthorization) but just provide a different OKTA URL  via environment variable.